src/secret-storage.ts

```diff
@@ -100,10 +100,12 @@ export interface PassphraseInfo {
  * Options for {@link ServerSideSecretStorageImpl#addKey}.
  */
 export interface AddSecretStorageKeyOpts {
-    pubkey?: string;
```

Comment:
This is some legacy field that was used for the deprecated `m.secret_storage.v1.curve25519-aes-sha2` algorithm.